### PR TITLE
Fix dashboard green button when tabbing

### PR DESCRIPTION
**Pull request by lavipr**
Pull Request for Issue #241 .

### Summary of Changes
Add missing element states focus and active to the quickicons containing "success" as classname. 

### Testing Instructions
On the Control Panel View tab to the quickicons within the "Update Checks"

### Expected result
The background-color and color should switch, so background-color gets green and color grey.
image

### Actual result
See Issue: https://github.com/joomla/backend-template/issues/241#issuecomment-496016120

### CODE REVIEW

**Comment by brianteeman** [comment]
Thank you very much for the detailed description

**Comment by brianteeman** [comment]
I can confirm that this PR fixes the issue.

(Hint : you should also check to ensure that quickicons containing "danger" or "warning" as classname do not have the same issues)

**Comment by lavipr** [comment]
Thanks :) Updated also for "danger" and "warning"

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 1]
```diff
@@ -191,8 +178,18 @@
         color: $danger-txt;
       }
 
-      &:hover {
+      &:hover,
+      &:focus,
+      &:active {
```
Comment (bembelimen) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: Perhaps you could use the hover/focus/active mixing from bootstrap here?

**Comment by lavipr** [comment]
thanks :) and updated

**Review by bembelimen — APPROVED** [review]

**Pull request merged**